### PR TITLE
[CX-2331]: Remove AROptionsArtistSeries feature flag

### PR DESCRIPTION
**Pull request by nickskalkin**
The type of this PR is: **chore**

Jira: [CX-2331]

### Description

Removes old AROptionsArtistSeries feature flag. The feature is already released for more than a month and the feature flag is not needed anymore.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I have added tests/stories for my changes, or my changes don't require testing/stories, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/main/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/main/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

### To the reviewers 👀

- [x] I would like at least one of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

Removes old AROptionsArtistSeries feature flag - Nikita

</details>

[CX-2331]: https://artsyproduct.atlassian.net/browse/CX-2331?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d427dffc40d35c3176e26efa0e17acef9bfb23ca
</p>

File: src/lib/Scenes/Artwork/Artwork.tsx [hunk 1]
```diff
@@ -65,9 +64,6 @@ export const Artwork: React.FC<ArtworkProps> = ({
   const [refreshing, setRefreshing] = useState(false)
   const [fetchingData, setFetchingData] = useState(false)
 
-  const artistSerieasEnabled = useFeatureFlag("AROptionsArtistSeries")
-  const artistSeriesEnabled = useFeatureFlag("AROptionsArtistSeries")
```
Comment (olerichter00) [inline src/lib/Scenes/Artwork/Artwork.tsx:69]: This is amazing!

**Review by olerichter00 — APPROVED** [review]
Thanks a lot 👍

**Comment by olerichter00** [comment]
#squashongreen

**Pull request merged**